api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -53,6 +53,10 @@ public interface WorkspaceMapper {
       target = "initialCredits.expirationEpochMillis",
       source = "dbWorkspace.creator",
       qualifiedByName = "getInitialCreditsExpiration")
+  @Mapping(
+      target = "initialCredits.extensionEpochMillis",
+      source = "dbWorkspace.creator",
+      qualifiedByName = "getInitialCreditsExpiration")
```

Comment:
this sets extension to the same value as expiration.  Is this intentional?

Reply:
Whoops, nope. I corrected it. Thank you for pointing it out.